### PR TITLE
[tmva] Add a new tutorial for TMVA RSofieReader class

### PR DESCRIPTION
**Pull request by lmoneta**
This Pull request add an example on using the new RSofieReader class

### CODE REVIEW

File: tutorials/tmva/TMVA_SOFIE_RSofieReader.C [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+/// ```
+///
+/// \macro_code
+/// \macro_output
+/// \author Lorenzo Moneta
+
+using namespace TMVA::Experimental;
+
+
+using namespace TMVA::Experimental;
```
Comment (sanjibansg) [inline tutorials/tmva/TMVA_SOFIE_RSofieReader.C:24]: This line is repeated. It's also there in line 24.

File: tutorials/tmva/TMVA_SOFIE_RSofieReader.C [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+   RSofieReader model("Higgs_trained_model.h5");
+
+   // the input shape for this model is a tensor with shape (1,7)
+
+   std::vector<float> input = {0.1,0.2,0.3,0.4,0.5,0.6,0.7};
+
+   // predict model on a single event (takes a std::vector<float>)
+
+   auto output = model.Compute(input);
```
Comment (sanjibansg) [inline tutorials/tmva/TMVA_SOFIE_RSofieReader.C:36]: This line gives an error. As inside the RSofieReader, the RModel object is named as `model`, thus probably here the model gets shadowed, thus here it should be named differently, maybe `RSofieReader reader()`.

**Review by sanjibansg — APPROVED** [review]
LGTM!

File: tutorials/tmva/TMVA_SOFIE_RSofieReader.C (hunk 1 above)
Comment (lmoneta) [reply tutorials/tmva/TMVA_SOFIE_RSofieReader.C:24]: Thank you! I will fix this!

**Pull request merged**